core/src/main/java/nu/yona/server/subscriptions/service/UserAnonymizedDto.java

```diff
@@ -25,22 +25,23 @@
 public class UserAnonymizedDto implements Serializable
 {
 	private static final long serialVersionUID = 5569303515806259326L;
-	private static final ZoneId DEFAULT_TIME_ZONE = ZoneId.of("Europe/Amsterdam");
 
 	private final UUID id;
 
 	private final LocalDate lastMonitoredActivityDate;
+	private ZoneId timeZone;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this value-based field transient so it is not included in the serialization of this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS3437)